src/pytorch_lightning/strategies/tpu_spawn.py

```diff
@@ -147,7 +149,7 @@ def distributed_sampler_kwargs(self) -> Dict[str, int]:
     @property
     def is_distributed(self) -> bool:
         # HOST_WORLD_SIZE is None outside the xmp.spawn process
-        return os.getenv(xenv.HOST_WORLD_SIZE, None) and self.world_size != 1
+        return os.getenv(xenv.HOST_WORLD_SIZE, None) is not None and self.world_size != 1
```

Comment:
Great! I also found this bug in #11098. A bit shorter and easier to read:
```suggestion
        return (xenv.HOST_WORLD_SIZE in os.environ) and self.world_size != 1
```